package com.java;

import java.lang.reflect.InvocationTargetException;

public class Q77 {

	static void m1(int[] a) {
		
	}
	public static void main(String[] args) {
		int a[]= {100,100};
		try {
		m1(a);
		}catch (IllegalArgumentException e) {
			System.err.println("Second");
		}
		catch(Exception e
				) {
			System.err.println("Third");
		}
	}

}
